.github/CODEOWNERS

```diff
@@ -88,8 +88,8 @@
 
 
 # App Feedback Reporting project
-/core/domain/app_feedback_report_*.py @BenHenning @vinitamurthi
-/core/controllers/incoming_app_feedback_report*.py @BenHenning @vinitamurthi
+/core/domain/app_feedback_report_*.py @BenHenning
```

Comment:
@vinitamurthi is it a temporary change or a permanent change? (If the first one then can you please follow [these instructions](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers#temporary-codeowner-transfer-process)?)

Reply:
This is a permanent change

Reply:
Thanks for the confirmation! :)